tests/js/spec/views/organizationGroupDetails/groupEventDetailsContainer.spec.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import OrganizationEnvironmentsStore from 'app/stores/organizationEnvironmentsStore';
+import GroupEventDetailsContainer from 'app/views/organizationGroupDetails/groupEventDetailsContainer';
+
+jest.mock('app/views/organizationGroupDetails/groupEventDetails', () => ({
+  GroupEventDetails: () => <div>GroupEventDetails</div>,
+}));
+
+describe('groupEventDetailsContainer', () => {
+  const org = TestStubs.Organization();
+  const globalSelection = TestStubs.GlobalSelection();
+
+  beforeEach(() => {
+    OrganizationEnvironmentsStore.init();
+  });
+
+  it('fetches environments', async function() {
+    MockApiClient.addMockResponse({
+      url: `/organizations/${org.slug}/environments/`,
+      body: [{id: 'id', name: 'name'}],
+    });
+    const wrapper = mount(
+      <GroupEventDetailsContainer
+        api={new MockApiClient()}
+        organization={org}
+        selection={globalSelection}
+      />
+    );
+    // should be in loading state
+    expect(wrapper.find('LoadingIndicator').exists()).toBe(true);
+    await tick();
+    await tick();
+    wrapper.update();
+    // should be loaded
+    expect(wrapper.find('LoadingIndicator').exists()).toBe(false);
+    expect(wrapper.find('GroupEventDetails').exists()).toBe(true);
+
+    // remounting will not rerender
+    const wrapper2 = mount(
```

Comment:
Oooh, here's a case where you can assert that the API endpoint only gets called once